lib/pure/uri.nim

```diff
@@ -316,18 +372,13 @@ proc `/`*(x: Uri, path: string): Uri =
   ## Contrary to the ``combine`` procedure you do not have to worry about
```

Comment:
`combine` should be a link to that proc:

```
Contrary to the `combine proc <#combine,Uri,Uri>`_ you do not have to worry about
```

Reply:
72ee0b8d6668d76073c11f8b86bb54772c248aff